travis/test_server.py

```diff
@@ -463,6 +463,25 @@ def main(argv=None):
         return 1
     elif counted_errors != expected_errors:
         return 1
+    # no test error, let's generate .pot and msgmerge all .po files
+    must_run_makepot = (
+        os.environ.get('MAKEPOT') == '1' and
+        os.environ.get('TRAVIS_REPO_SLUG', '').startswith('OCA/') and
```

Comment:
I see this is still a bit coupled for OCA only repos. Isn't there any way to change it?

Reply:
Well, this method is used elsewhere in MQT.

Besides, I'm convinced that we should stop trying to make MQT work for other use cases than OCA. We could simplify it vastly by focusing on OCA needs here, and putting reusable stuff in standalone packages that people can easily compose in their own CI workflow (such as is done for pylint-odoo and click-odoo-makepot).

One good candidate is a package specialized to manipulate manifest files (@legalsylvain suggested to call it `manifestoo`). This package would provide an library and CLI to explore Odoo manifests such as finding installable addons, or listing dependencies of a list of addons, etc. But that's another story.

Reply:
OK, let's continue this way right now and if anybody needs this for other repo, he/she can propose an isolation.